src/client-websocket.ts

```diff
@@ -0,0 +1,11 @@
+declare let window: any;
+let _global = typeof global !== 'undefined' ? global : (typeof window !== 'undefined' ? window: {});
+
+import * as NodeSocket from 'ws';
```

Comment:
Hmm.  dist/client.js still pulls in dist/client-websocket.js which has a `require('ws')` written in it (as opposed to in the old websocket module whose lib/client.js literally doesn't mention require).  Are you sure this will work properly when webpacked? I admit I forget how I tested this a few weeks ago...